test/Sema/lifetime_attr.swift

```diff
@@ -28,7 +28,7 @@ func invalidDuplicateLifetimeDependence1(_ ne: borrowing NE) -> NE {
 
 class Klass {}
 
-@lifetime(x) // expected-error{{invalid use of lifetime dependence on an Escapable parameter with consuming ownership}}
+@lifetime(borrow x) // expected-error{{invalid use of borrow dependence with consuming ownership}}
```

Comment:
Would it be possible to make this `_borrow` instead of `borrow` so that the argument of `@lifetime` parses like a normal expression? We would really like to avoid adding more custom parsing rules for attributes.

Reply:
I don't think we will want an `_borrow` when `@lifetime` is no longer experimental.

Reply:
Do we have an accepted spelling of `borrow` that we could use here? What I care about is that the argument to `@lifetime` parses as a normal expression.

Reply:
The current "borrow" operator is spelled as `_borrow` under an experimental flag. If that is spelled as `borrow` it would parse as an expression. Would that be fine ?

Reply:
Yes, if we changed `_borrow` to `borrow` then it would be fine.

Reply:
Although in the near future, we will want to support `@lifetime(targetName: [borrow] sourceName1, sourceName2 ...)`.

Reply:
I think it would. `targetName` would parse as an argument label and `borrow sourceName1` or `sourceName1` would be the arguments, both of which are valid expressions.